deluge/ui/console/main.py

```diff
@@ -10,165 +10,58 @@
-        "(?<![\\-a-z])(-[a-zA-Z0-9])": "{!red!}%s{!input!}",
-        # "(\-[a-zA-Z0-9])": "{!red!}%s{!input!}",
-        "--[_\\-a-zA-Z0-9]+": "{!green!}%s{!input!}",
-        "(\\[|\\])": "{!info!}%s{!input!}",
-
-        "<tab>": "{!white!}%s{!input!}",
-        "[_A-Z]{3,}": "{!cyan!}%s{!input!}",
-
-        "<download-folder>": "{!yellow!}%s{!input!}",
-        "<torrent-file>": "{!green!}%s{!input!}"
-
-    }
-
-    def __init__(self,
-                 indent_increment=2,
-                 max_help_position=24,
-                 width=None,
-                 short_first=1):
-        optparse.IndentedHelpFormatter.__init__(
-            self, indent_increment, max_help_position, width, short_first)
-
-    def _format_colors(self, string):
-        def r(repl):
-            return lambda s: repl % s.group()
-
-        for key, replacement in self.replace_dict.items():
-            string = re.sub(key, r(replacement), string)
-
-        return string
-
-    def format_usage(self, usage):
-
-        return _("{!info!}Usage{!input!}: %s\n") % self._format_colors(usage)
-
-    def format_option(self, option):
-        result = []
-        opts = self.option_strings[option]
-        opt_width = self.help_position - self.current_indent - 2
-        if len(opts) > opt_width:
-            opts = "%*s%s\n" % (self.current_indent, "", opts)
-            opts = self._format_colors(opts)
-            indent_first = self.help_position
-        else:  # start help on same line as opts
-            opts = "%*s%-*s  " % (self.current_indent, "", opt_width, opts)
-            opts = self._format_colors(opts)
-            indent_first = 0
-        result.append(opts)
-        if option.help:
-            help_text = self.expand_default(option)
-            help_text = self._format_colors(help_text)
-            help_lines = optparse.textwrap.wrap(help_text, self.help_width)
-            result.append("%*s%s\n" % (indent_first, "", help_lines[0]))
-            result.extend(["%*s%s\n" % (self.help_position, "", line)
-                           for line in help_lines[1:]])
-        elif opts[-1] != "\n":
-            result.append("\n")
-        return "".join(result)
-
-
-class OptionParser(optparse.OptionParser):
-    """subclass from optparse.OptionParser so exit() won't exit."""
-    def __init__(self, **kwargs):
-        optparse.OptionParser.__init__(self, **kwargs)
+    def format_help(self):
+        """
+        Differs from ArgumentParser.format_help by adding the raw epilog
+        as formatted in the string. Default bahavior mangles the formatting.
+
+        """
+        try:
+            # Handle epilog manually to keep the text formatting
+            epilog = self.epilog
+            self.epilog = ""
+            help = super(ConsoleCommandParser, self).format_help()
+            if epilog is not None:
+                help += epilog
+            self.epilog = epilog
+            return help
+        except Exception as ex:
```

Comment:
Hmm...not really.  Agree that it should not catch Exception. Tbh, it could just be part of dev testing and may not be needed now.